resources/qml/pages/LoginPage.qml

```diff
@@ -13,7 +13,7 @@ Page {
         }
 
         PageHeader {
-            title: qsTr("Login")
+            title: qsTr("Wellcome to Shmoose")
```

Comment:
You changed the title of the page. Not a bad idea. But you used a wrong spelling. Must be 'Welcome'.